java/runtime/src/main/java/org/ray/runtime/objectstore/RayObjectProxy.java

```diff
@@ -0,0 +1,12 @@
+package org.ray.runtime.objectstore;
+
+public class RayObjectProxy {
```

Comment:
nit: maybe `RayObjectBuffer`? proxy looks like something that would run for a long time, e.g. ObjectStoreProxy

Reply:
Changed to `NativeRayObject`.